apps/api-catalogue/components/ServiceCard/ServiceCard.spec.tsx

```diff
@@ -7,6 +7,7 @@ import {
   TypeCategory,
   DataCategory,
 } from '@island.is/api-catalogue/consts'
+import { ApiService } from '@island.is/api/schema'
```

Comment:
This seems to be unused import?

Reply:
this unit test has been rewritten on main and should be fine if merged correctly.